src/coreclr/scripts/exploratory.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python3
+#
+## Licensed to the .NET Foundation under one or more agreements.
+## The .NET Foundation licenses this file to you under the MIT license.
+#
+##
+# Title: exploratory.py
+#
+# Notes:
+#
+# Script to execute exploratory tool on a platform and return back the repro
+# issues they found.
+#
+################################################################################
+################################################################################
+
+import shutil
+import argparse
+from os import linesep, listdir, path, walk
+from os.path import isfile, join, getsize
+import os
+from os import listdir
+from coreclr_arguments import *
+from superpmi_setup import run_command, copy_directory, set_pipeline_variable
```

Comment:
It's worrying that you're importing from superpmi scripts. Maybe there should be some "utility" scripts that both superpmi and exploratory can import from.

Reply:
Yes...eventually as we discussed in https://github.com/dotnet/runtime/pull/53750/files#r646958941. I have opened https://github.com/dotnet/runtime/issues/59559 to track it.